README.md

```diff
@@ -1,27 +1,114 @@
-# WipPlatformFe
+# Setup
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 17.0.8.
+Clone the repository and install packages with: `npm install`
 
-## Development server
+# General rules
 
-Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The application will automatically reload if you change any of the source files.
+When writing code we follow the SOLID principles and try to keep the code clean.<br>
+More on SOLID: https://wearecommunity.io/communities/epam-poland/articles/1190<br>
+More on how to keep the code clean: https://gist.github.com/wojteklu/73c6914cc446146b8b533c0988cf8d29
 
-## Code scaffolding
+# Project structure
 
-Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
+We use the following project structure:
+![image](https://github.com/WIP-Work-In-Progress/wip-platform-fe/assets/143722910/cce925b8-3498-49e0-9eba-8f707ba82c34)<br>
+In the **shared** directory, we follow the SCAM pattern.<br>
+More on the SCAM pattern: https://angular-training-guide.rangle.io/modules/module-scam-pattern<br>
+More on the main parts of an angular application: https://atejszerska.pl/blog/1/details
 
-## Build
+# Designs
 
-Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory.
+We use figma to design our UI-s which are then used in the project.<br>
+Our figma: https://www.figma.com/files/project/179730628/Team-project?fuid=1321955424622355736
 
-## Running unit tests
+# Contributing
+	
+If you want to contribute, we require that you abide by the following conventions:
 
-Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.io).
+## Use the english language
```

Comment:
Use English language <- more gramatically correct (:

Reply:
I think both are correct, but after some thought i would change it to just "Use English" as its the simplest and less formal